pkg/ocm/helpers.go

```diff
@@ -405,3 +409,33 @@ func (c *Client) CheckRoleExists(orgID string, roleName string, awsAccountID str
 	}
 	return true, existingRole[1], nil
 }
+
+func GetLatestROSACliVersion() (string, error) {
```

Comment:
The implementation here assumes that the format and url of the downloads page will never change. If they ever do it will cause all clients to start failing.

Reply:
Perhaps a fallback generic message "We could not determine the latest version. Go to https://mirror.openshift..."